mucommander-core/src/main/java/com/mucommander/ui/theme/ThemeManager.java

```diff
@@ -57,14 +64,18 @@
  * @author Nicolas Rinaudo
  */
 public class ThemeManager {
-	private static final Logger LOGGER = LoggerFactory.getLogger(ThemeManager.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ThemeManager.class);
 
     // - Class variables -----------------------------------------------------------------
     // -----------------------------------------------------------------------------------
     /** Path to the user defined theme file. */
     private static       AbstractFile userThemeFile;
     /** Default user defined theme file name. */
     private static final String       USER_THEME_FILE_NAME             = "user_theme.xml";
+
+    /** The location of available fonts list as getting them from OS takes long time (impacts startup) */
+    private static final String       AVAILABLE_FONT_CACHE             = "available_fonts_cache.cache";
```

Comment:
did you consider storing this cache in the snapshot file instead?

Reply:
> did you consider storing this cache in the snapshot file instead?

nope, it didn't came to my mind back then